docs/uptime-guide/overview.asciidoc

```diff
@@ -1,11 +1,56 @@
 [role="xpack"]
 [[uptime-overview]]
-== Overview
+== Elastic Uptime overview
 
-The Elastic Uptime solution built on the Elastic Stack.
-It allows you to monitor the availability and response time of software services and applications in real time.
+The Elastic Uptime solution is built on the Elastic Stack.
+Elastic Uptime allows you to monitor the availability and response times of applications and services in real time and to detect problems before they affect users.
+
+Elastic Uptime is based on {heartbeat-ref}/index.html[Heartbeat], an open source uptime monitor.
```

Comment:
Maybe better to say "Elastic's Uptime solution consists of the pairing of the Uptime Kibana app with the Heartbeat agent."

Reply:
I've removed this sentence completely, because it adds nothing, and all the components of Uptime, Elasticsearch, Kibana and Heartbeat, are mentioned succinctly in the next paragraph.